model/src/main/kotlin/DependencyNavigator.kt

```diff
@@ -82,6 +82,19 @@ interface DependencyNavigator {
         matcher: DependencyMatcher = MATCH_ALL
     ): Map<String, Set<Identifier>>
 
+    /**
+     * Return a set with the [Identifier]s of packages that are dependencies of package with the given [packageId] in
+     * the given [project]. Starting from [project], the dependency graph is searched for the package in question;
```

Comment:
I found the first sentence difficult to read, maybe: "...that are dependencies of **the** package with the given [packageId] in the **dependency tree of** the given [project]."